lib/challenge.js

```diff
@@ -19,7 +19,15 @@
   // - park
 
 // Answer Starts Here
-
+'Calendar'
+Description: our app uses a calendar to keep track of appointments.
+Every 'calendar' has
+- an appointment name
+- an appointment date
+- an appointment location  (ok, that is three, I tend to use calendars a bit).
```

Comment:
This is how I view a calendar. As an app that creates and tracks appointments. What am I missing here?